lib/AST/Builtins.cpp

```diff
@@ -177,7 +176,7 @@ getBuiltinFunction(Identifier Id, ArrayRef<Type> argTypes, Type ResType,
                              /*GenericParams=*/nullptr,
                              paramList,
                              TypeLoc::withoutLoc(ResType), DC);
-  FD->computeType(Info);
+  (void)FD->getInterfaceType();
```

Comment:
None of the new calls to (void)fn->getInterfaceType() should be necessary. Were you not able to remove them?

Reply:
I'm bad at rebasing, and when I was writing this up this morning I must have built on top of a bad revision.  I think these can all go away now.